src/main/scala/com/datastax/sparkstress/SparkCassandraStress.scala

```diff
@@ -193,6 +203,31 @@ object SparkCassandraStress {
     }
   }
 
+  def unpackAnnotatedNumeric(num: String): Long = {
+
+    if (num forall Character.isDigit) {
+      num.toLong
+    } else {
+      val numericPortion = num.init
+      assert(numericPortion forall Character.isDigit)
+
+      val numericAnnotation = num.toLowerCase.last
+      assert(Character.isLetter(numericAnnotation))
+
+      try {
```

Comment:
why the multiplier is it not a part of enum itself? 
imagine you would like to pretty print it or sth - will you repeat this logic?